Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Stage 1: Build Stage
-ARG PYTHON_VERSION=3.8
+ARG PYTHON_VERSION="default"
```

Comment:
Using 'default' as the value for `PYTHON_VERSION` is not recommended. It's better to specify a specific version to ensure consistency across different environments.